.github/dependabot.yml

```diff
@@ -8,6 +8,28 @@ updates:
     open-pull-requests-limit: 1
     schedule:
       interval: "weekly"
+    # https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuring-dependabot-version-updates#example-disabling-version-updates-for-some-dependencies
```

Comment:
Why do we want to discard minor updates, e.g., `cid` or `bls-signature`?

Reply:
`cid` is dependency of `fvm*` crates, minor version updates make them incompatible

e.g.
```toml
[[package]]
name = "fvm_ipld_blockstore"
version = "0.2.0"
source = "registry+https://github.com/rust-lang/crates.io-index"
checksum = "417f52f6915b9f9a68de8462e1cf46f14a2c16420f484b8d2066873de2ffe420"
dependencies = [
 "anyhow",
 "cid",
 "multihash",
]

[[package]]
name = "fvm_ipld_encoding"
version = "0.4.0"
source = "registry+https://github.com/rust-lang/crates.io-index"
checksum = "90608092e31d9a06236268c58f7c36668ab4b2a48afafe3a97e08f094ad7ae50"
dependencies = [
 "anyhow",
 "cid",
 "fvm_ipld_blockstore",
 "multihash",
 "serde",
 "serde_ipld_dagcbor",
 "serde_repr",
 "serde_tuple",
 "thiserror",
]
```